src/support/rgw_support_bundle

```diff
@@ -28,6 +28,9 @@ import argparse
 from cortx.utils.errors import BaseError
 from cortx.utils.process import SimpleProcess
 from cortx.utils.conf_store.conf_store import Conf, MappedConf
+from src.const import (
+    CONFIG_PATH_KEY, LOG_PATH_KEY, COMPONENT_NAME,
+    RGW_CONF_FILE, CRASHDUMP_DIR)
```

Comment:
RGW_CONF_FILE also we can make as CONF_FILE only, avoid RGW name in vars, wherever we can